extensions/dbtCloud/src/runs_list_items.tsx

```diff
@@ -0,0 +1,35 @@
+import { List, Icon, ActionPanel, Action } from "@raycast/api";
+import { RunModel } from "./types";
+
+interface JobRunsListItemProps {
+  result: RunModel;
+}
+
+export const JobRunsListItem = ({ result }: JobRunsListItemProps): JSX.Element => {
+  const item = result;
+  return (
+    <List.Item
+      id={item.id.toString()}
+      title={item.job.name}
+      accessoryTitle={item.finished_at_humanized}
```

Comment:
Is deprecated, could be like this now (since we support multiple 😄 ):

```suggestion
      accessories={[{text: item.finished_at_humanized}]}
```